### ISSUE TITLE
No scrollbars in Chrome OSX for content

### ISSUE THREAD

**Issue by draganescu** [WordPress/gutenberg#16966]
**Describe the bug**
In Safari and in Firefox there is a scrollbar that shows up on scroll (the default setting in OSX) except for Chrome where there is no scrollbar unless I select "always" from system settings.

**Comment by jasmussen** [comment]
Is this unique to the block editor or can you reproduce this in other scrolling containers in Chrome?

**Comment by draganescu** [comment]
No scrolling containers in Gutenberg show the scrollbars. But if I exit Gutenberg, everythingin WP Admin shows them as expected.

**Comment by jasmussen** [comment]
Indeed, I can reproduce. That is most curious. It appears to potentially be related to a Chrome bug: https://bugs.chromium.org/p/chromium/issues/detail?id=927066. Here's more discussion on it: https://support.google.com/chrome/thread/3101009?hl=en

Specifically it happens because the containers on which the scrollbar is not present, are `position: fixed;`, and as you mentioned, only in Chrome.

This is almost certainly a Chrome bug, and a frustrating one at that. I found sort of a workaround; if you apply this:

```
.edit-post-visual-editor {
	overflow: auto;
}
```

![hack](https://user-images.githubusercontent.com/1204802/63674146-2f363d00-c7e6-11e9-9b9e-23b7e9cc8201.gif)

The problem is the same for the sidebar scrolling, which also leverages position: fixed;

It appears to work again with no negative effects. HOWEVER, this is pretty sensitive territory for such CSS, and inded this is the container that themes alone are supposed to style. So most of all I'd love some more thoughts on this. It's frustrating to have to create hopefully temporary hacks to accommodate Chrome running in a specific Mac configuration, but at the same time it's quite disruptive if you land in that specific group.

So to summarize, this bug appears on Chrome MacOS if you have auto-hiding scrollbars enabled. And it appears to primarily be related to containers that are supposed to scroll, _and_ have `position: fixed;` applied. The workaround is to move the overflow rule to an inner container that does not use `position: fixed;`. However it's not that simple, as this basic codepen seems to work: https://codepen.io/joen/pen/VwZpQJB?editors=1100

I have verified that this is unfortunately also a problem in Chrome Canary (version 78, so it's not fixed upstream yet.), and even the new Chromium based Microsoft Edge.

I will investigate further, but would love additional eyes on it.

**Comment by jasmussen** [comment]
Progress:

![progress](https://user-images.githubusercontent.com/1204802/63675584-6b1ed180-c7e9-11e9-8eea-d2a1dfe47d6a.gif)

When I delete these two elements:

```
<div class="components-notice-list components-editor-notices__dismissible"></div>
<div class="components-snackbar-list components-editor-notices__snackbar"></div>
```

Everything works as it should. Not yet sure why, but we're closer.

**Comment by jasmussen** [comment]
Okay, #17188 fixes it. But it is exotic!

**Comment by jasmussen** [comment]
As an alternative to the exotic PR referenced above, the Chrome issue has been acknowledged and assigned. May be worth waiting a bit more to see if there's progress on that ticket, and if there is close this as an upstream issue.

**Comment by karmatosed** [comment]
As far as design feedback goes on this, so long as fixed 👍 Removing that label for now.

**Comment by draganescu** [comment]
This is somewhat improved in Chrome 80, and Gutenberg master, the sidebar does show a scrollbar but the content still doesn't.

**Comment by skorasaurus** [comment]
Hi, 
This appears to have been fixed thanks to https://bugs.chromium.org/p/chromium/issues/detail?id=997607 
(Ref: https://github.com/cloudflare/cloudflare-docs-engine/issues/137) 

If this is still an issue, could you please reopen the issue.
